packages/goto/src/index.js

```diff
@@ -56,13 +56,13 @@ module.exports = ({ timeout, ...deviceOpts }) => {
 
   const goto = async (
     page,
-    { url, media, adblock = true, headers = {}, waitFor = 0, disableAnimations = true, disableJavaScript = false, ...args }
+    { url, mediaType, adblock = true, headers = {}, waitFor = 0, animations = true, javascript = false, ...args }
```

Comment:
It seems a bit confusing to me that passing “JavaScript=true” will actually disable JavaScript in the page.

Reply:
The script only needs to disabled if it is running, and if script is running the status of the **javascript** variable must be true.
So it makes perfect sense to me that setting it true will eventually disable it.

Reply:
resolved at https://github.com/microlinkhq/browserless/pull/130/commits/9ff0a3f3ade9942e1e5dd353c25d235ab44f5001